.github/workflows/push.yml

```
@@ -68,7 +66,29 @@ jobs:
           push: true
           tags: ghcr.io/${{ github.repository }}:${{ needs.branch_meta.outputs.sha }}
           labels: ${{ steps.docker_meta_img.outputs.labels }}
-
+      - name: Docker meta Service Name (file storage)
```

After each Step or job an empty line is need for code style
```suggestion

      - name: Docker meta Service Name (file storage)
```